src/pricecypher/models/datasets.py

```diff
@@ -15,6 +15,7 @@ class Scope:
     representation: Optional[str] = field(compare=False)
     name_dataset: str = field(compare=False)
     name_human: Optional[str] = field(compare=False)
+    name_human_times_volume: Optional[str] = field(compare=False)
```

Comment:
Might be good to also add the `volume_multiplied_constant` to the ScopeConstantTransaction model

Reply:
Added